### PR TITLE
entity mappings

### PR DESCRIPTION
**Pull request by Kahzerx**
more passive entity mappings

### CODE REVIEW

**Comment by Kahzerx** [comment]
oman, it keeps pushing all the commits, gotta fix it

**Comment by Copetan** [comment]
these commits 😭

File: mappings/1.7.4#14w02a.tinydiff [hunk 1]
```diff
@@ -8222,11 +8222,11 @@ c	net/minecraft/unmapped/C_5553933
 	m	()Z	m_9771828		isSearchingSpawnPoint
 	m	()Lnet/minecraft/unmapped/C_1713911;	m_0329262		getVillages
 c	net/minecraft/unmapped/C_5565812
-	f	I	f_9049662		f_9049662
-	f	I	f_6621006		f_6621006
-	m	(IZ)V	m_5833021		m_5833021
-		p	1			p_1
-		p	2			p_2
+	f	I	f_9049662		forcedAge
+	f	I	f_6621006		happyTicksRemaining
```
Comment (SpaceWalkerRS) [inline mappings/1.7.4#14w02a.tinydiff:8226]: this is getting a little too real

**Review by SpaceWalkerRS — APPROVED** [review]

File: mappings/1.7.4#14w02a.tinydiff (hunk 1 above)
Comment (Kahzerx) [reply mappings/1.7.4#14w02a.tinydiff:8226]: xd

**Pull request merged**